README.rst

```diff
@@ -33,7 +33,7 @@ Requirements
 ------------
 
 * Redis.
-* Python 2.6, 2.7 or 3.3.
+* Python 2.7 or 3.3+.
```

Comment:
@nichochar I fixed the python 2.6 issues in https://github.com/pinterest/snappass/pull/19/commits/00f6964a90ae29a6de6bff8e3295d7e93ebb055c
Relatively small glitches... though it's annoying not to have the latest assertion methods from 2.7+ !
Can't wait for 2.6 support to be dropped ;)